src/lib/sendPushNotification.ts

```diff
@@ -0,0 +1,57 @@
+import webNotification from 'simple-web-notification';
+import { INotification } from '../types';
+import { IS_CORDOVA, IS_EXTENSION, IS_WEB } from './environment';
+
+/**
+ * Schedules a web notification.
+ *
+ * @param {Partial<INotification>} notification - The notification to schedule.
+ */
+export function scheduleWebNotification(notification: Partial<INotification>) {
+  try {
```

Comment:
I think there is no need to wrap this block with `try...catch` because both `showNotification` and `requestPermission` are not throwing anything.